modules/learn/pages/services-and-indexes/services/index-service.adoc

```diff
@@ -36,10 +36,14 @@ Each index is created on one bucket only; but multiple indexes may be created on
 
 The Index Service can be configured to use either _standard_ or _memory-optimized_ storage:
 
-* _Standard_: The supervisor process persists to disk all changes made to individual indexes.
+Standard::
+The supervisor process persists to disk all changes made to individual indexes.
 Each index gets a dedicated file.
-Writes can either be _append-only_  or _write with circular reuse_, depending on the write mode selected for global secondary indexes.
-* _Memory-Optimized_: Indexes are saved in-memory.
+* In Couchbase Server Community Edition, writes can either be _append-only_  or _write with circular reuse_, depending on the write mode selected for global secondary indexes.
+* In Couchbase Server Enterprise Edition, compaction is handled automatically.
+Memory-Optimized::
+Only available in Couchbase Server Enterprise Edition.
+Indexes are saved in-memory.
```

Comment:
It would be good to clarify here that indexes are not only in-memory. A lot of users have asked about this in the past. We could add "A snapshot of the index is maintained on disk, to permit rapid recovery if node-failures are experienced.".

Reply:
Done